src/button/view/button.jsx

```diff
@@ -151,6 +151,12 @@ export default class Button extends Component {
                     [`${prefix}icon-alone`]: count === 1,
                     [child.props.className]: !!child.props.className,
                 });
+
+                if ('size' in child.props) {
+                    log.warning(
+                        'The size of Icon will not take effect, when Icon is the [direct child element] of Button(<Button><Icon size="xs" /></Button>), use <Button iconSize={}> or <Button><div><Icon size="xs" /></div></Button> instead of.'
```

Comment:
` <Icon size="${child.props.size}" /> will not take effect, use <Button iconSize={}> or <Button><div><Icon size="xs" /></div></Button> instead of.